downloader/src/main/kotlin/vcs/Cvs.kt

```diff
@@ -150,17 +150,15 @@ class Cvs : VersionControlSystem(), CommandLineTool {
                         decimals.dropLast(1).last() == "0"
             }
 
-            override fun listRemoteBranches(): List<String> {
-                return listSymbolicNames().mapNotNull { (name, version) ->
+            override fun listRemoteBranches() =
```

Comment:
prefer explicit return type here

Reply:
This clearly returns a list of names, and names are usually strings, so to me the return type is obvious.